test/api/__snapshots__/api.test.js.snap

```diff
@@ -0,0 +1,177 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I always thought Jest snapshots were useful only for UI components (i.e. to detect unexpected rendering changes).  Do we really need to enable (and store) them for the API tests?  (I last used Jest over a year ago, so best practices could have changed since then.)

Reply:
They are, but in the case I'm using them as a shorthand to write a lot of assertions regarding how many times, in which order something has been executed.
I get your point though, but maybe this helps: I did have a careful look at all the generated snapshots so I didn't simply have the suite generate them for me without me having to ever care about the test again. I also intentionally created "named mocks" I.e. mocks that return a string with their name when executed, so they show up in the snapshots.